pfcli/configurator/deployment.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (C) 2021 FriendliAI
+
+"""PeriFlow Deployment Configurator."""
+
+from __future__ import annotations
+
+import json
+from dataclasses import dataclass
+from typing import Any, Dict, Type
+
+from pfcli.configurator.base import IO, Configurator, InteractiveConfigurator
+from pfcli.service import EngineType
+from pfcli.utils.format import secho_error_and_exit
+
+INFERENCE_SERVER_CONFIG = """
+# Inference server type config
+inference_server_type:
+  name:
+  repo:
+  tag:
+"""
+
+ORCA_CONFIG = """
+# Orca engine config
+orca_config:
+  max_batch_size:
+  max_token_count:
+  kv_cache_size:
+"""
+
+CKPT_CONFIG = """
+# Orca ckeckpoint config
+ckpt_config:
+  version:
+  catagory:
+  data_type:
+"""
```

Comment:
What is `version` of this config?

Reply:
I just copied it from `config.py`. Can you suggest the correct value?

<img width="870" alt="image" src="https://user-images.githubusercontent.com/17061663/229971233-0e42cbd0-8c28-4f87-8314-ce479470a993.png">